makeflow/src/makeflow.c

```diff
@@ -1599,6 +1626,19 @@ int main(int argc, char *argv[])
 		makeflow_gc_method = MAKEFLOW_GC_ALL;
 	}
 
+	if(wrapper_umbrella && wrapper_umbrella->spec) {
+		debug(D_MAKEFLOW_RUN, "setting dag_node->umbrella_spec...\n");
+		struct dag_node *cur;
+		cur = d->nodes;
+		while(cur) {
+			dag_node_set_umbrella_spec(cur, wrapper_umbrella->spec);
```

Comment:
Okay. If you really want, I can remove it for now. 
But the next PR will definitely have these code again.